articles/Introduction-to-time-series/index.md

```diff
@@ -0,0 +1,60 @@
+
+![Simple](https://github.com/jamessandy/engineering-education/blob/new-article/articles/introduction-to-time-series/hero.jpg)
+
+In Machine learning, time series is a very crucial aspect but is often overlooked. They are useful in forecasting, stock market analysis even forex because it is the time component that makes it challenging to handle. This article will give a friendly introduction to time series.
+
+#### Tabel of Content
+1. Introductions to Time series
+2. Real-life applications of Time series
+3. Time-series Forecasting & Time-series Analysis
+4. Time Series Forecasting methods
+5. Components of Time Series
+6. Conclusion
+
+Time series can be defined as a sequence or series of data points that are ordered in time. In traditional machine learning, the dataset is a collection of observations, and predictions are made based on the new data, which likely will not be known until the future. The future is what is predicted, but all the previous observations are put into considerations. In time series, the dataset is different. Time-series adds a definite order dependence between observations.
+
+Time series are assumed to be generated at a spaced timing. When the data in time series are timed are regular, they are called **regular time series** and when they are not at a regular interval, they are called **irregular time series**
+
+#### Real-life applications of Time series
+1. Can be used to predicting the closing price of the stock at the end of the day
+2. Used in Predicting the number of product sales
+3. Forecasting the birth or death rate at a hospital
+4. Forecasting the number of passengers a bus terminal will have 
+5. Used to forecast the unemployment rate of a city or state
+
+#### Time-series Forecasting
+Time series forecasting is fitting a model on a historical dataset, which is the training set, and then using it to predict a future occurrence, which is the test set. The performance of the time series model is rated based on its future prediction. The whole idea is that our model observes all the previous data and understands the foundational process in the series. Secondly, it predicts based on its understanding of the foundational process of the historical data.
+
+#### Time-series Analysis
+Time series analysis can be defined as getting meaningful insight and characteristics of the data to comprehend it better. Time series is beneficial in making predictions. It understands the scope of nature and uses it for future forecasting.
+
+#### Time Series Forecasting methods 
+Now let’s look at four time-series forecasting methods and have a brief description of them.
+1. Autoregressive Integrated Moving Average (ARIMA): This method combines both the Autoregression and Moving Average model. ARIMA models the next step in the sequence as a linear function based on the difference in observation and residual errors at previous steps. This method works best for univariate time series with trend and without seasonal components.
```

Comment:
A person reading this would not be aware of auto-regression and moving average models. Can you explain a bit more about these? Like the intuition behind them. And then, you may guide the reader to articles that explore these topics in depth.